tests/beta/test_default_tensor_tf.py

```diff
@@ -739,7 +749,7 @@ def test_tf_interface_classical_diff(self, cost_with_decomposition, interface, d
         assert np.allclose(res.numpy(), self.expected_grad, atol=tol, rtol=0)
 
 
-    def test_error_classical_diff_torch(self, torch_support, tol):
+    def test_error_classical_diff_torch(self, torch_support, tol, rep):
```

Comment:
Added `rep` simply as a convenience due to parametrization, could create a new class as well

Reply:
yeah that's one annoying thing about pytest